platform/barefoot/sonic-platform-modules-ingrasys/debian/sonic-platform-ingrasys-s9280-64x.postrm

```diff
@@ -0,0 +1,42 @@
+# Automatically added by dh_systemd_start
+if [ -d /run/systemd/system ]; then
+    systemctl --system daemon-reload >/dev/null || true
+    fi
+# End automatically added section
+# Automatically added by dh_installinit
+if [ "$1" = "purge" ] ; then
+    update-rc.d s9280-64x-monitor remove >/dev/null
+    update-rc.d bfn remove >/dev/null
+    update-rc.d qsfp-monitor remove >/dev/null
+fi
+
+
+# In case this system is running systemd, we make systemd reload the unit files
+# to pick up changes.
+if [ -d /run/systemd/system ] ; then
+    systemctl --system daemon-reload >/dev/null || true
+fi
+# End automatically added section
+# Automatically added by dh_systemd_enable
+if [ "$1" = "remove" ]; then
+    if [ -x "/usr/bin/deb-systemd-helper" ]; then
+        deb-systemd-helper mask s9280-64x-monitor.service >/dev/null
+        deb-systemd-helper mask bfn.service >/dev/null
+        deb-systemd-helper mask qsfp-monitor.service >/dev/null
+    fi
+fi
+
+if [ "$1" = "purge" ]; then
+    if [ -x "/usr/bin/deb-systemd-helper" ]; then
+        deb-systemd-helper purge s9280-64x-monitor.service >/dev/null
+        deb-systemd-helper unmask s9280-64x-monitor.service >/dev/null
+        deb-systemd-helper purge bfn.service >/dev/null
+        deb-systemd-helper unmask bfn.service >/dev/null
```

Comment:
please do not add, this is automatically generated file.

Reply:
We will remove all the automatically generated files, thanks.